scss/civicrm/contact/pages/_batch.scss

```diff
@@ -221,14 +192,11 @@
         }
 
         .crm-form-date-wrapper {
-          width: 370px;
+          display: inline-block;
+          width: 255px;
```

Comment:
This change is a bit worrying. Why it is exactly 255px, can't it be calculated somehow from other variables? At least, let's move that to a separate variable.

Reply:
Will leave a comment